src/requestManager.cpp

```diff
@@ -682,6 +682,7 @@ bool CRequestManager::IsNodePingAcceptable(CNode *pfrom)
     if (pfrom->nPingUsecTime < ACCEPTABLE_PING_USEC)
         return true;
 
+#if 0 // TODO: this needs to be calculated periodically (rarely) and outside of any locks
```

Comment:
is this PR to consider WIP till you implement the above TODO?

Reply:
good catch, this is an optimization that is unrelated to this PR. I'll remove it from here.